easybuild/easyconfigs/q/QIIME2/QIIME2-2024.10.1-foss-2023b-amplicon.eb

```
@@ -0,0 +1,274 @@
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['d4caa104432cf1c6b9e689de770ba68eef0de18a2dd7b32a74b70cf0cbfa297d'],
+    }),
+    ('q2-quality-control', '2024.10.0', {
+        'source_urls': ['https://github.com/qiime2/q2-quality-control/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['a00e0fa0659d8aef45b0e8f9ac0aec6a9073c8193af4493f1fd7d851090b746b'],
+    }),
+    ('q2-quality-filter', '2024.10.0', {
+        'source_urls': ['https://github.com/qiime2/q2-quality-filter/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['18b7198d15dfe71cabff744d3d1644d6f0433132594bdce3cbfec6046bd1f51b'],
+    }),
+    ('q2-sample-classifier', '2024.10.0', {
+        'source_urls': ['https://github.com/qiime2/q2-sample-classifier/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['1f499cb2e02a7cab18f1d0ac0863b03fccca141b46f944facfb965dedfb96e3e'],
+    }),
+    ('q2-taxa', '2024.10.0', {
+        'preinstallopts': "make all && ",
+        'source_urls': ['https://github.com/qiime2/q2-taxa/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['7e71ce9461c57fb5d884320a920cf7de9f778f0efac6b83647028b5eca6a7457'],
+    }),
+    ('q2-types', '2024.10.0', {
+        'source_urls': ['https://github.com/qiime2/q2-types/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['ef253689bc2c3a1a7c8f0f41e6503919efe1e44fd635ab79d2dc7be1d80b3c84'],
+    }),
+    ('q2-vsearch', '2024.10.0', {
+        'source_urls': ['https://github.com/qiime2/q2-vsearch/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['87b2f7476f41a2761d4cbfdddd3c5111249a92a89aac92b516ea22bbe8e6e89a'],
+    }),
+    ('q2templates', '2024.10.0', {
+        'source_urls': ['https://github.com/qiime2/q2templates/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['e937bb8b2a46ff774f6813ec3448fb921cba20cc72dc757d4c9b3621502eb8f8'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/qiime'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = [
+    "qiime --help",
+    "export CONDA_PREFIX='' && qiime info",
+    "qiime tools validate "
+    "%(installdir)s/lib/python%(pyshortver)s/site-packages/q2_sample_classifier/tests/data/vaw.qza",
+]
+
+modextrapaths = {'CONDA_PREFIX': ''}
```

better to set this via `modextravars`, as is done in https://github.com/easybuilders/easybuild-easyconfigs/pull/22918:
```suggestion
modextravars = {'CONDA_PREFIX': '%(installdir)s'}